.github/workflows/build-matrix.yml

```diff
@@ -217,9 +217,24 @@ jobs:
         if: runner.os != 'Windows'
         run: |
           cmake --build build/Radium-Engine --parallel --config ${{ matrix.build-type }} --target check
-      - name: Run CoreExample as a build and install test
-        if: runner.os != 'Windows'
+          cmake --build build/Radium-Engine --parallel --config ${{ matrix.build-type }} --target Install_CoreExample
+          ./build/Radium-Engine/examples/CoreExample/CoreExample
+          ./install/bin/CoreExample
+      - name: Run unit tests on windows
+        if: runner.os == 'Windows'
         run: |
+          #Qt might be set by qt action
+          set PATH="$GITHUB_WORKSPACE/build/Radium-Engine/src/Core";\
+          "$GITHUB_WORKSPACE/build/Radium-Engine/src/Engine";\
+          "$GITHUB_WORKSPACE/build/Radium-Engine/src/Gui";\
+          "$GITHUB_WORKSPACE/build/Radium-Engine/src/Headless";\
+          "$GITHUB_WORKSPACE/build/Radium-Engine/src/IO";\
+          "$GITHUB_WORKSPACE/build/Radium-Engine/src/PluginBase";\
+          "$GITHUB_WORKSPACE/glfw_install/lib/cmake/glfw3/";\
+          "${{ env.ext-dir }}/lib";\
+          "%PATH%"
+          echo %PATH%
+          cmake --build build/Radium-Engine --parallel --config ${{ matrix.build-type }} --target check
           cmake --build build/Radium-Engine --parallel --config ${{ matrix.build-type }} --target Install_CoreExample
           ./build/Radium-Engine/examples/CoreExample/CoreExample
           ./install/bin/CoreExample
```

Comment:
I suggest to use HeadlessExample instead of CoreExample. The former will test more functionalities than the later.  
HeadlessExample will test Radium::Core, Radium::Engine, Radium::IO and Radium::Headless in a single run and the produced image (the Radium logo if no scene parameter is given) might be used as a badge ;p 
On windows and macos, just replace CoreExample by HeadlessExample, on Linux, this will require to install and use xvfb 
 - install using `apt-get install -y --no-install-recommends xvfb xauth`
 - run using xvfb-run -a examples/HeadlessExample/HeadlessExample

Reply:
After verification, no need to instal xvfb as it is installed on ubuntu-latest github virtual environment (https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md)

Reply:
Let's keep CoreExample for now, new tests (with HeadlessExample and image compare) welcom in futur PR.